From 1cb19f0e26f2af6349cb4b588338367f2b9ed95d Mon Sep 17 00:00:00 2001
From: Andrew Bays <abays@redhat.com>
Date: Thu, 14 Dec 2017 13:30:25 -0500
Subject: [PATCH 27/53] require libyajl2 to build sysevent plugin, part 2

(cherry picked from commit 477ddba49ba5da255cb71cee69132dbdd83a417d)
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 42258aeb..df173687 100644
--- a/configure.ac
+++ b/configure.ac
@@ -6663,7 +6663,7 @@ AC_PLUGIN([snmp_agent],          [$with_libnetsnmpagent],   [SNMP agent plugin])
 AC_PLUGIN([statsd],              [yes],                     [StatsD plugin])
 AC_PLUGIN([swap],                [$plugin_swap],            [Swap usage statistics])
 AC_PLUGIN([synproxy],            [$plugin_synproxy],        [Synproxy stats plugin])
-AC_PLUGIN([sysevent],            [$with_libyajl],           [Syslog event statistics])
+AC_PLUGIN([sysevent],            [$plugin_sysevent],        [Syslog event statistics])
 AC_PLUGIN([syslog],              [$have_syslog],            [Syslog logging plugin])
 AC_PLUGIN([table],               [yes],                     [Parsing of tabular data])
 AC_PLUGIN([tail],                [yes],                     [Parsing of logfiles])
-- 
2.17.2

